app/containers/Initializer.js

```diff
@@ -39,10 +41,13 @@ class Initializer extends React.Component {
       initCurrency,
       initWallets,
       initChannels,
+      initAutopayList,
     } = this.props
     initTickers()
     initLocale()
     initCurrency()
+    initAutopayList()
```

Comment:
I'd probably rename this to `initAutopay` as this method should do all autopay initialisation code. Currently it only initialises the list, but it may do more later.